components/inputgroup/index.css

```diff
@@ -69,6 +75,14 @@ governing permissions and limitations under the License.
     var(--spectrum-actionbutton-m-texticon-min-width) / 2 -
       var(--spectrum-alias-ui-icon-chevron-size-100) / 2
   );
+
+  /* todo: when we add t-shirt sizing, this will be wrong ❤️ */
+  --spectrum-textfield-icon-inline-end-override: calc(var(--spectrum-alias-infieldbutton-full-height-m) + var(--spectrum-global-dimension-size-100));
+}
+
+/* hack: use var so it doesn't disappear */
+.🤫 {
```

Comment:
is this emoji intentional?

Reply:
Heh, yeah. I didn't want to define a classname like `.x` and end up screwing up some client page. Unfortunately, we need this so that the build system doesn't strip this variable.

Note that we're using CSS custom properties to change things between components -- this is cleaner than targeting selectors that would be very deep in some cases. It is a new approach, however, and it would be nice if we could make that hack less dirty.

Reply:
hmmm 🤔

Reply:
It works, haha. We can definitely rename it to `.____temp` or something, but I thought an emoji would be less likely to be used lol. Who knows.